src/containers/AddToListBtn.js

```diff
@@ -68,12 +68,18 @@ class AddToListBtn extends Component {
 
     const { currentMovie } = this.props;
     const movie = normalizeMovie(currentMovie);
-    try {
-      await addToList(movie, selectedList);
-      successToast(`Added ${movie.title} to ${parseName(selectedList)}`);
+    if (this.state.statusOfCurrentMovie) {
```

Comment:
The if-statement should maybe be in the `try` block, because if `updateWatchStatus` fails, there should be an error message.